lib/logger_json_file_backend.ex

```diff
@@ -63,6 +63,7 @@ defmodule LoggerJSONFileBackend do
         true ->
           json_encoder.encode!(Map.merge(%{level: level, uuid: UUID.uuid4(), message: (msg |> IO.iodata_to_binary), time: format_time(ts)}, take_metadata(md, keys, triming))) <> "\n"
         false ->
+          metadata = take_metadata(md, keys, triming)
```

Comment:
```
variable "metadata" is unused
```
Could you check it?

Reply:
removed, and verified that `mix test test` still passes